.gitignore

```diff
@@ -24,3 +24,5 @@ spec/dummy/public/uploads/*
 .rubocop.yml
 
 *.gem
+
+vendor/
```

Comment:
@jamesmk I didn't see it in the existing code, so I though it was added with something I did. You want it?